src/Mafia/Init.hs

```diff
@@ -205,6 +246,7 @@ instance FromJSON MafiaState where
       MafiaState <$>
         o .: "source-dependencies" <*>
         o .: "cabal-file" <*>
+        o .: "lock-file" <*>
```

Comment:
> Whoops. Missed the version bump above. Do we support old versions as well?

Nope, we don't need to, it's not compulsory information, just saves 5-10second on a `mafia build`